compiler/src/codegen/runtime_errors.re

```diff
@@ -4,5 +4,6 @@ open Sexplib.Conv;
 [@deriving sexp]
 type grain_error =
   | IndexOutOfBounds
+  | IndexNonInteger
```

Comment:
Does the order of these affect anything? Does it make it breaking?

Reply:
Nope!